### DISCUSSION TITLE
Where is the current documentation to create the same experience as in the Demo?

### DISCUSSION THREAD

**Discussion by manuganji** [steven-tey/novel#293] [Q&A] [answered]
I went through code, documentations and the website. I ran into errors. For example, `EditorCommand` doesn't take `children` prop at least that's the typescript error being thrown.

**Comment by andrewdoro** [comment]
Hey here you can find the implementation for the demo https://github.com/steven-tey/novel/blob/main/apps/web/components/tailwind/editor.tsx
Strange that you get this error, I'll look into it.

**Reply by manuganji** [reply]
I went through this code. `novel/extensions` is not exported.

> Cannot find module 'novel/extensions' or its corresponding type declarations.

> Cannot find module 'novel/plugins' or its corresponding type declarations.

**Reply by andrewdoro** [reply, marked as answer]
Hey thanks for the input, it was type mismatch with react.
I've released a new version here, can you try it ?
[0.2.2](https://www.npmjs.com/package/novel/v/0.2.2)

**Reply by manuganji** [reply]
Type error corresponding to `EditorCommand` is now gone. Thank you. :)

But `novel/extensions` error is still present.

**Reply by robmck1995** [reply]
Sorry to revive this but I am also seeing the issue that `novel/extensions` cannot be found. I'm sure I'm just doing something stupid but it's not clear to me what that is

**Comment by imranhirey** [comment]
thank you guys i really appreciated your hard word you really saved me time
